packages/rocketchat-message-attachments/client/messageAttachment.coffee

```diff
@@ -49,7 +49,10 @@ Template.messageAttachment.helpers
 			return Meteor.user()?.settings?.preferences?.collapseMediaByDefault is true
 
 	time: ->
-		return moment(@ts).format(RocketChat.settings.get('Message_TimeFormat'))
+		if moment(@ts).format(RocketChat.settings.get('Message_DateFormat')) == moment(Date.now()).format(RocketChat.settings.get('Message_DateFormat'))
```

Comment:
Ah, sorry, I didn't realize it was coffeescript. In that case, we use `is` instead of `==`